libraries/abstractions/lwip_osal/src/sys_arch.c

```diff
@@ -0,0 +1,712 @@
+/*
```

Comment:
Verified that this file is a duplicate of [the original](https://github.com/aws/amazon-freertos/blob/master/libraries/3rdparty/lwip/src/portable/arch/sys_arch.c) with minor formatting changes.